docs/maui/layouts/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Layouts - .NET MAUI Community Toolkit
+author: VladislavAntonyuk
+description: The .NET MAUI Community Toolkit extends .NET MAUI layouts.
+ms.date: 04/21/2022
+---
+
+# Layouts
```

Comment:
Just a general note from the maui docs plus a link to the rest.

```suggestion
# Layouts

.NET Multi-platform App UI (.NET MAUI) layout classes allow you to arrange and group UI controls in your application. Choosing a layout class requires knowledge of how the layout positions its child elements, and how the layout sizes its child elements. In addition, it may be necessary to nest layouts to create your desired layout.

For further information on Layouts please refer to the [.NET MAUI documentation](/dotnet/maui/user-interface/layouts/).
```